app/views/works/_standard_form.html.erb

```diff
@@ -327,22 +327,22 @@
   <% end %><!-- end of fields for chapters -->
 
   <fieldset class="create">
-    <legend><%= ts('Post') %></legend>
+    <legend><%= ts("Post") %></legend>
     <ul class="actions">
       <% unless @work.new_record? || @work.posted? %>
         <li>
-          <%= submit_tag ts('Save Without Posting'), name: 'save_button' %>
+          <%= submit_tag ts("Save Without Posting"), name: 'save_button' %>
```

Comment:
The various `name`s for buttons down here can all have their quotation marks changes, too.